app/src/main/java/com/example/ohthmhyh/activities/UpdateHabitActivity.java

```diff
@@ -47,7 +47,8 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.alert_addhabit);
+        setContentView(R.layout.activity_update_habit);
+        this.setTitle("Add/Update your Habit");
```

Comment:
```suggestion
        this.setTitle("View Habit");
```

Reply:
When you click "add a habit", it will show View Habit too. I think it's best to leave it like this so we can worry abt other things.

Reply:
We might also have this activity be responsible for showing another person's Habit in their feed. I might just change it to "Habit" to make it very general.